client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -223,6 +275,17 @@ class Upload extends Component {
 			upgradeUrl = `/plans/${ siteSlug }`;
 		}
 
+		if ( isEligibleForHostingTrial ) {
+			/* translators: %(planName)s the short-hand version of the Business plan name */
+			title = translate(
+				'Start your free %(planName)s plan trial to access the theme install features',
+				{
+					args: { planName: getPlan( PLAN_BUSINESS )?.getTitle() ?? '' },
+				}
+			);
+			upgradeUrl = '#';
```

Comment:
Do we absolutely need to pass the `#`?

Reply:
If we don't the upsell internals gonna add plans url

Reply:
unless we change the internals

Reply:
Seems like it's trickier to change the internals, but I only glanced over the code.

Reply:
Yeah I didn't want to go down that rabbit hole

Reply:
Sounds good 👍